types/jsprintmanager/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "dtslint/dt.json" ,
+    "rules": {
```

Comment:
Is there any specific need for turning off this rules?

Reply:
I removed the rules, and just kept the one for the interfaces